CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.0] - 2017-08-14
+### Added
+- `sign_widevine_zip` only extracts the files we need to sign, and appends the sigfiles to the original zipfile.
```

Comment:
I should say, it's always a pleasure to read such detailed release notes, especially when they appear at the top of the PR! 👍